apps/src/applab/designElements/ColorPickerPropertyRow.jsx

```diff
@@ -24,6 +24,15 @@ export default class ColorPickerPropertyRow extends React.Component {
     window.removeEventListener('mousedown', this.handlePageClick);
   }
 
+  componentWillReceiveProps(nextProps) {
+    const {initialValue} = nextProps;
+    if (this.props.initialValue !== initialValue) {
+      this.setState({
+        value: initialValue
+      });
+    }
+  }
```

Comment:
Most react components I've dealt with handle property changes, but we are inconsistent.

In this case, if you change the screen's theme, which changes the screen's background color, even with an explicit call to `renderDesignWorkspace()`, the component will re-render with the out-of-date "state value" instead of the updated "property" value.

Given that the changes take effect immediately on the design mode page, I'm not sure it makes sense to have the the "state value" persist after a "property" change.

Let me know if you have thoughts...

Reply:
Just wondering why we need `state.value` at all here - usually it's for some intermediate "editing but not saved" state, so that's my first guess. It'd be nice if we didn't though.

Reply:
that's a good point - I don't actually see why it is needed.

Reply:
Worth removing if not needed?